File: webservices/src/main/java/fieg/modulos/cadastro/bancos/dto/BancoDTO.java
package fieg.modulos.cadastro.bancos.dto;


import lombok.Data;

import java.time.LocalDateTime;


@Data
public class BancoDTO {

    private Integer id;
    private String numero;
    private String nome;
    private String abreviatura;
    private LocalDateTime dataInclusao;
    private Integer idOperadorInclusao;
    private LocalDateTime dataAlteracao;
    private Integer idOperadorAlteracao;

}

File: web/src/models/DTOs/AgenciaBancoConta/BancoDTO.ts
export interface BancoDTO {
  id: number;
  numero: string;
  nome: string;
  abreviatura: string;
  dataInclusao: string;
  idOperadorInclusao: number | null;
  dataAlteracao: string;
  idOperadorAlteracao: number | null;
}
